src/XeroPHP/Remote/Collection.php

```diff
@@ -17,6 +17,16 @@ public function addAssociatedObject($parent_property, Model $object)
         $this->_associated_objects[$parent_property] = $object;
     }
 
+    /**
+     * Return whether or not the Collection is 0
+     *
+     * @return bool
+     */
+    public function empty()
+    {
+        return !sizeof($this);
```

Comment:
Could you please consider using `count()` instead (`sizeof()` is an alias of `count()` and can be ambiguous due to its behaviour in other c-based languages)

Reply:
Sure! Going to do that.